mimic/test/test_core.py

```diff
@@ -29,10 +29,13 @@ def test_from_plugin_includes_all_plugins(self):
         :class:`MimicCore`, the nova and loadbalancer plugins are included.
         """
         core = MimicCore.fromPlugins(Clock())
-        plugin_apis = set((nova_plugin.nova, loadbalancer_plugin.loadbalancer,
-                           swift_plugin.swift, queue_plugin.queue,
-                           maas_plugin.maas, rackconnect_v3_plugin.rackconnect,
-                           nova_plugin.nova_control_api, glance_plugin.glance))
+        plugin_apis = set((
```

Comment:
We should probably change this list to be one-per-line so that adds and removes are more clearly visible.